bitnami/thanos/templates/receive/ingress.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.receive.enabled .Values.receive.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "common.names.fullname" . }}-receive
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+  annotations:
+    {{- if .Values.receive.ingress.certManager }}
+    kubernetes.io/tls-acme: "true"
+    {{- end }}
+    {{- range $key, $value := .Values.receive.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  rules:
+    {{- if .Values.receive.ingress.hostname }}
+    - host: {{ .Values.receive.ingress.hostname }}
+      http:
+        paths:
+          - path: {{ .Values.receive.ingress.path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
+            pathType: {{ .Values.receive.ingress.pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (printf "%s-%s"  (include "common.names.fullname" .) "receive") "servicePort" "http" "context" $)  | nindent 14 }}
+    {{- end }}
+    {{- range .Values.receive.ingress.extraHosts }}
+    - host: {{ .name | quote }}
+      http:
+        paths:
+          - path: {{ default "/" .path }}
+            {{- if eq "true" (include "common.ingress.supportsPathType" $) }}
+            pathType: {{ default "ImplementationSpecific" .pathType }}
+            {{- end }}
+            backend: {{- include "common.ingress.backend" (dict "serviceName" (printf "%s-%s"  (include "common.names.fullname" .) "receive") "servicePort" "http" "context" $) | nindent 14 }}
+    {{- end }}
+  {{- if or .Values.receive.ingress.tls .Values.receive.ingress.extraTls .Values.receive.ingress.hosts }}
+  tls:
+    {{- if or .Values.receive.ingress.secrets .Values.receive.ingress.tls }}
+    - hosts:
+        {{- range .Values.receive.ingress.extraHosts }}
```

Comment:
Sorry, maybe I explained so bad, we would like to have both

```
- {{ .Values.receive.ingress.hostname }}
{{- range .Values.receive.ingress.extraHosts }}
- {{ .name | quote }}
{{- end }}
```

Reply:
Thanks for patients. I have fixed it.

Reply:
Cool